webapp/src/ui/Button.tsx

```
@@ -0,0 +1,16 @@
+import type { IconDefinition } from "@fortawesome/fontawesome-svg-core";
+import { Icon } from "./Icon";
+
+export interface ButtonProps {
+    size: 'xs' | 'sm' | 'md' | 'lg',
+    icon: IconDefinition,
+}
+
+
+export function Button({ size, icon }: ButtonProps) {
+    return(
+        <button className={`btn-${size}`}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)